Podverse/AddToPlaylistViewController.swift

```diff
@@ -193,7 +197,7 @@ extension AddToPlaylistViewController:UITableViewDelegate, UITableViewDataSource
                     if let cell = self.tableView.cellForRow(at: indexPath) as? PlaylistTableViewCell {
                         
                         if let itemCount = itemCount {
-                            cell.itemsTotal.text = "Items: " + String(describing: itemCount)
+                            cell.itemCount.text = "Items: " + String(describing: itemCount)
```

Comment:
Not a huge deal but The describing initializer is not very widely used. Remember you can unwrap non string values like this: ''Items: \(itemCount)"